config/default_config.yml

```diff
@@ -11,6 +11,11 @@
 ######################################################################################
 ######################################################################################
 
+# Advanced settings, you should only touch these if you really know what you're doing
+ADVANCED_CRCON_SETTINGS:
+  # The number of concurrent connections to the game server CRCON will open
```

Comment:
Bit misleading maybe, as it's the number of connection per process, so the total is much higher than that.
Maybe also add that the sweet spot is 10-20 and that below 4 we don't recommend using the automod as there might be significant delays in the Game view.